subjects/devops/comparator/README.md

```diff
@@ -0,0 +1,21 @@
+## comparator
+
+### Instructions
+
+Create a script `comparator.sh` which will verify if the first given argument is bigger, smaller or equal than the second given argument.
+You must print te output like the example in usage.
+
```

Comment:
If no checks about the argument passed are required, shouldn't this be specified in the subject? Something like
> the script will assume that it will be always run with two numbers as arguments

Reply:
Done

Reply:
The subject should include some information on what should happen when there is an error